deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -70,6 +70,9 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip"  title="Customize user data">User data configs:</label>
+                            {% if is_pinterest %}
+                                <a href={{user_data_config_settings_wiki}}><span class="glyphicon glyphicon-question-sign"></span></a>
```

Comment:
can you add target="_blank" after href so the wiki will show up in a new tab?

Reply:
good point, thanks, updated.